assets/js/draw.js

```diff
@@ -57,3 +57,8 @@ function draw(e) {
   ctx.lineTo(mousePos.x, mousePos.y);
   ctx.stroke();
 }
+
+download_img = function(el) {
+  var image = canvas.toDataURL("image/jpg");
```

Comment:
Convert jpg to jpeg.
After downloading the canvas the background is black 
It would be nice if it is white. Ping me if need any help.

Reply:
okay. Thank You @991rajat !!!

Reply:
![Screenshot from 2021-03-27 11-21-09](https://user-images.githubusercontent.com/29520476/112711461-8d5d6180-8eee-11eb-9a53-18b749258604.png)
Added white background.